index.html

```diff
@@ -1039,62 +965,23 @@ <h2>
         Dependencies
       </h2>
       <p>
-        <code><dfn data-cite="HTML/dom.html#document">Document</dfn></code>,
-        <code><dfn data-cite=
-        "HTML/window-object.html#window">Window</dfn></code>,
-        <code><dfn data-cite=
-        "HTML/workers.html#dedicatedworkerglobalscope">DedicatedWorkerGlobalScope</dfn></code>,
-        <dfn data-cite="HTML/browsers.html#active-document">active
-        document</dfn>, <dfn data-cite=
-        "HTML/webappapis.html#global-object">global object</dfn>,
-        <dfn data-cite="HTML/browsers.html#browsing-context">browsing
-        context</dfn>, <code><dfn data-cite=
-        "HTML/webappapis.html#eventhandler">EventHandler</dfn></code> are
-        defined in [[HTML]].
-      </p>
-      <p>
-        <dfn data-cite="DOM#abortsignal-add">add the following abort steps</dfn>
-        is defined in [[DOM]].
-      </p>
-      <p>
-        <dfn data-cite="WEBIDL#aborterror"><code>AbortError</code></dfn>,
-        <dfn data-cite="WEBIDL#notsupportederror"><code>NotSupportedError</code></dfn>,
-        <dfn data-cite="WEBIDL#notallowederror"><code>NotAllowedError</code></dfn>,
-        <dfn data-cite="WEBIDL#dfn-DOMException"><code>DOMException</code></dfn>
-        are defined in [[WEBIDL]].
-      </p>
-      <p>
-        The terms <dfn data-cite="PERMISSIONS#query-a-permission">query a permission</dfn>,
-        <dfn data-cite="PERMISSIONS#request-permission-to-use">requesting permission to use</dfn>,
-        <dfn data-cite="PERMISSIONS#permission-revocation-algorithm">revocable</dfn>,
-        <dfn data-cite="PERMISSIONS#permission-descriptor-type">permission descriptor type</dfn>,
-        <dfn data-cite="PERMISSIONS#dom-permissiondescriptor-name">permission name</dfn>,
-        <dfn data-cite="PERMISSIONS#permission-state">permission state</dfn>,
-        <dfn data-cite="PERMISSIONS#powerful-feature">powerful feature</dfn>,
-        <dfn data-cite="PERMISSIONS#dom-permissionstate-denied"><code>denied</code></dfn>,
-        <dfn data-cite="PERMISSIONS#dom-permissionstate-granted"><code>granted</code></dfn>,
-        <dfn data-cite="PERMISSIONS#dom-permissionstate-prompt"><code>prompt</code></dfn> and
-        <dfn data-cite="PERMISSIONS#dictdef-permissiondescriptor">PermissionDescriptor</dfn>
-        are defined in [[PERMISSIONS]].
-      </p>
-      <p>
-        <dfn data-cite="ECMASCRIPT#sec-promise-objects"><code>Promise</code></dfn> and
-        <dfn data-cite="ECMASCRIPT#sec-object-internal-methods-and-internal-slots">internal slots</dfn>
-        are defined in [[ECMASCRIPT]].
+        <dfn data-cite="HTML/webappapis.html#global-object">global object</dfn>
```

Comment:
@sidvishnoi, I wasn't able to get these ones to link. Any suggestions? Same with the IDL ones below.

Reply:
`<dfn class="externalDFN">term</dfn>` should work

Reply:
Just recording that I tried:

`<dfn class="externalDFN">global object</dfn>`
`<dfn class="externalDFN" data-xref-for="">global object</dfn>`

both failed tho... we identified a server issue.